app/controllers/concerns/mission_control/jobs/job_filters.rb

```diff
@@ -9,7 +9,14 @@ module MissionControl::Jobs::JobFilters
 
   private
     def set_filters
-      @job_filters = { job_class_name: params.dig(:filter, :job_class_name).presence, queue_name: params.dig(:filter, :queue_name).presence }.compact
+      @job_filters = { job_class_name: params.dig(:filter, :job_class_name).presence, queue_name: params.dig(:filter, :queue_name).presence,
+                       finished_at_start: params.dig(:filter, :finished_at_start).presence, finished_at_end: params.dig(:filter, :finished_at_end) }.compact
+      finished_at_timezoned_filters
```

Comment:
I think this should be in this method, inlined, and have another helper method to do the parsing.

Reply:
i pushed some changes about this, I could move `date_with_time_zone` method to the helpers and include here

Reply:
hi @rosa

just wanted to ping you on this one, no worries however, I know you are probably super bussy with other stuff